sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -3561,6 +3561,26 @@ public void Cfg_Default_Statement_First()
             defaultBlock.SuccessorBlock.Should().Be(exitBlock);
         }
 
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_Ref_Arg_Should_Be_Last_Instruction1()
```

Comment:
these 2 tests should not be inside the `switch` region, but rather in a region of their own

TBH we could split this file in multiple files using the `partial class` feature, the same way we do with the SonarSecurity InstructionFactory tests; but that would be in a different PR